packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -184,12 +184,11 @@ async function verifyRequestState(options: AuthenticateRequestOptions, token: st
  * Let the next rule for UatMissing to fire if needed
  */
 export const isSatelliteAndNeedsSyncing: InterstitialRule = options => {
-  const { clientUat, isSatellite, searchParams, secretKey, apiKey } = options;
+  const { sessionUat, clientUat, isSatellite, searchParams } = options;
 
-  const key = secretKey || apiKey;
-  const isDev = isDevelopmentFromApiKey(key);
+  const isSignedOutOrExpired = !clientUat || clientUat === '0' || !sessionUat;
 
-  if (isSatellite && (!clientUat || clientUat === '0') && !hasJustSynced(searchParams) && !isDev) {
+  if (isSatellite && isSignedOutOrExpired && !hasJustSynced(searchParams)) {
```

Comment:
❓ Based on the PR description, this change will result in syncing more frequently in production and less regularly in development. 

Why don't we preserve the same frequency across dev and prod and achieve better parity between the two environments? That is, the highest frequency should be applied to both instance types.

Reply:
Maybe I phrased it incorrectly. 

Before we had: 
- Highest frequency is on every hard refresh (only for dev)
- Lowest frequency is only if user is logged out (only for prod)

After this PR:
- Intermediate frequency: (parity between dev and prod)
  - Session is expired
  - Logged out